Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -487,7 +487,7 @@ extension KMConversationViewController {
             switch result {
             case .success(let conversationFeedback):
                 print("feedback submit response success: \(conversationFeedback)")
-                guard conversationFeedback.feedback != nil else { return }
+                guard let conversationFeedback = conversationFeedback.feedback else { return }
```

Comment:
We've to keep these changes. These changes were done after the SPM PR was merged, so SPM and other changes shouldn't be removed.